openclaw_hooks/nima-recall-live/ladybug_recall.py

```
@@ -91,274 +169,181 @@ def text_search(query: str, limit: int = MAX_RESULTS) -> List[Dict]:
-    except Exception as e:
-        print(f"[ladybug_recall] Get related error: {e}", file=sys.stderr)
-        return [], 0
-
 # =============================================================================
-# COMPATIBILITY LAYER (drop-in replacement for lazy_recall_v3)
+# COMPATIBILITY LAYER
 # =============================================================================
 
 def lazy_recall(query: str, 
                  query_embedding: Optional[List[float]] = None,
                  who: Optional[str] = None,
```

Ruff (0.15.1)

[warning] Unused function argument: `who`

(ARG001)